<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEmbeddedEye</class>
 <widget class="QMainWindow" name="QEmbeddedEye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEmbeddedEye</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Main Actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_startPanoramicCam">
         <property name="text">
          <string>Start Pan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_startFovealCam">
         <property name="text">
          <string>Start Foveal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_doTracking">
         <property name="text">
          <string>Do Tracking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_moveMotors">
         <property name="text">
          <string>Move Motors</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Status Information</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Execution Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ProcessingTime">
           <property name="text">
            <string>Proc Time</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Focus Position:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_XYPos">
           <property name="text">
            <string>( X , Y )</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCamera_Settings"/>
    <addaction name="actionTracking_Settings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Project"/>
   </widget>
   <widget class="QMenu" name="menuOpen_Display">
    <property name="title">
     <string>Open Display</string>
    </property>
    <addaction name="actionPanoramic"/>
    <addaction name="actionFoveal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOpen_Display"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCamera_Settings">
   <property name="text">
    <string>Camera Settings</string>
   </property>
  </action>
  <action name="actionAbout_Project">
   <property name="text">
    <string>About Project</string>
   </property>
  </action>
  <action name="actionTracking_Settings">
   <property name="text">
    <string>Tracking Settings</string>
   </property>
  </action>
  <action name="actionPanoramic">
   <property name="text">
    <string>Panoramic</string>
   </property>
  </action>
  <action name="actionFoveal">
   <property name="text">
    <string>Foveal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
